scripts/live_test.sh

```diff
@@ -0,0 +1,158 @@
+        exit 1
+    fi
+}
+
+ensure_b2k_is_disconnected() {
+    echo "ensure b2k is disconnected successfully"
+    ## see if b2k pods are present, future iterations check the image name
+    RESTORE_POD_NAME_FOR_DISCONNECT=$(kubectl get pods -n todo-app -o custom-columns=NAME:.metadata.name | grep -P "restore")
+    echo "restore pod name after disconnection:$RESTORE_POD_NAME_FOR_DISCONNECT"
+    if [[ -z $RESTORE_POD_NAME_FOR_DISCONNECT ]]; then
+        echo "B2K restore pod is not present after disconnection"
+        exit 0
+    else 
+        echo "B2K restore pod is still present after disconnection"
+        exit 1
+    fi
+}
+
+start_b2k() {
+    echo 'Starting Bridge'
+    ## & for parallel execution , && for sequential execution
+    ../../../src/dsc/bin/Debug/net6.0/$OS/dsc connect --service stats-api --namespace todo-app --local-port 3001 --control-port 51424 --use-kubernetes-service-environment-variables -- npm run start & b2kPID=$!
+    sleep 30
+    echo "b2k process ID is $b2kPID"
+}
+
+start_minikube_tunnel() {
+    echo 'Starting minikube tunnel'
+    minikube tunnel  > /dev/null 2>&1 & tunnelPID=$!
+    sleep 10
+    echo 'verifying if minikube tunnel works'
+    curl $(minikube service frontend -n todo-app --url)/api/stats
+}
+
+set_up_stats_api() {
+    echo 'cd into stats-api folder'
+    cd samples/todo-app/stats-api
+    echo 'npm install'
+    npm i
+}
+
+dotnet_publish_for_b2k() {
+    echo "dotnet publishing dsc"
+    dotnet publish src/dsc/dsc.csproj -c Debug -r $OS --self-contained true 
+    dotnet publish src/endpointmanager/endpointmanager.csproj -c Debug -r $OS
+}
+
+# Check JQ Processor and download if not present
+check_jq_processor_present() {
+    check_if_exists jq
+    if [[ $result != 0 ]]; then
+        install_tool jq
+    fi
+    jqversion=$(jq --version)
+    log INFO "Locally installed JQ Processor version is $jqversion"
+}
+
+check_kubectl_present() {
+    check_if_exists kubectl
+    if [[ $result != 0 ]]; then
+        install_tool kubectl
+    fi
+    kubectlversion=$(kubectl version --client=true -o json | jq ".clientVersion.gitVersion")
+    log INFO "Locally installed kubectl version is $kubectlversion"
+}
+
+check_if_exists() {
+    log INFO "checking if $1 exists"
+    if ! [[ -x "$(command -v $1)" ]]; then
+        log INFO "Error: $1 is not installed." >&2
+        result=1
+    else 
+        result=0
+    fi
+}
+
+log() {
+    local level=$1
+    shift
+    echo "$(date -u $now) - $level - $*"
+}
+
+start_live_test() {
+    echo "Starting live testing for $OS"
+    check_kubectl_present
+    check_jq_processor_present
+
+    dotnet_publish_for_b2k
+    set_up_stats_api
+    start_minikube_tunnel
+    #set bridge as dev environment to run live test because some of image tags might noe be available in production
+    export BRIDGE_ENVIRONMENT='dev'
+    start_b2k
+
+    validate_b2k_is_running
+
+    stop_b2k
+
+    #ensure_b2k_is_disconnected
```

Comment:
Why is this one commented?

Reply:
@sabbour I tried adding them as single script but didn't work out so commented them. I will keep them for reference and clean up in next PR.

Reply:
By the way I took care of this test as another step in Github action (verify.sh as separate script)
![image](https://user-images.githubusercontent.com/105889062/233191393-f3f45eec-98ff-48d8-9fa9-ee4e859c6bf6.png)

Reply:
This will be cleaned up in upcoming PR. Thanks.